radixdlt/babylon-wallet-android
# ABW-2553 - Just one behaviour (SUPPLY_FLEXIBLE) is shown for xrd.

## Description
For XRD we add only one behaviour (SUPPLY_FLEXIBLE) to match what it is on iOS.

## How to test

1. Go to XRD resource and verify there is only one behaviour (SUPPLY_FLEXIBLE)
2. For all other tokens behaviours remain the same

## Screenshot

<img src="https://github.com/radixdlt/babylon-wallet-android/assets/108684750/b984e047-20df-400c-b789-43accccbd19f" width="300">

## PR submission checklist
- [x] I have checked xrd details view to see if behaviours are correct
